README.md

```diff
@@ -36,11 +36,20 @@ Via `homebrew`:
 $ brew tap twilio/brew && brew install twilio
 ```
 
+## Link the Twilio CLI Infra Plugin
+
+To get started, run `npm install` to install the plugin package dependencies. Afterwards, run the following command:
+
+```sh-session
+$ twilio plugins:link <path_to_your_local_repo>
+```
+
+You can now start using the `twilio infra` commands on your machine.
+
 ## Usage
 
 ```sh-session
-$ twilio plugins:install kaiquelupo/plugin-twilio-infra
-$ twilio --help infra
+$ twilio infra --help
```

Comment:
Strangely enough both `twilio --help infra` and `twilio infra --help` works fine :-) Either way I'm fine with that

Reply:
I prefer `twilio infra --help` as I think this is the convention Twilio is following? Also what I use to work with other CLI tools.